src/Test/Perf/util/DownloadUtilities.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.IO;
```

Comment:
Both `DownloadUtilities.cs` and `Download_util.csx` can be deleted since bootstrap.bat now does the downloading